ocs_ci/ocs/ui/helpers_ui.py

```diff
@@ -95,6 +100,101 @@ def format_locator(locator, string_to_insert):
+
+    """
+    base_ui_obj = PageNavigator(setup_ui)
+
+    ocp_version = get_ocp_version()
+    pvc_loc = locators[ocp_version]["storage_class"]
+
+    base_ui_obj.navigate_storageclasses_page()
+    logger.info("Create Storage Class")
+    base_ui_obj.do_click(pvc_loc["create-sc"])
+    logger.info("Storage Class Name")
+    sc_name = create_unique_resource_name(
+        resource_description="test", resource_type="storageclass"
+    )
+    base_ui_obj.do_send_keys(pvc_loc["sc-name"], f"{sc_name}")
+    logger.info("Storage Class Description")
+    base_ui_obj.do_send_keys(pvc_loc["sc-description"], "this is a test storage class")
+    logger.info("Storage Class Reclaim Policy")
+    base_ui_obj.do_click(pvc_loc["reclaim-policy"])
+    base_ui_obj.do_click(pvc_loc["reclaim-policy-delete"])
+    logger.info("Storage Class Provisioner")
+    base_ui_obj.do_click(pvc_loc["provisioner"])
+    base_ui_obj.do_click(pvc_loc["rbd-provisioner"])
+    logger.info("Storage Class Storage Pool")
+    base_ui_obj.do_click(pvc_loc["storage-pool"])
+    base_ui_obj.do_click(pvc_loc["ceph-block-pool"])
+    if encryption:
+        logger.info("Storage Class with Encryption")
+        base_ui_obj.do_click(pvc_loc["encryption"])
+        logger.info("Checking if 'Change connection details' option is available")
+        conn_details = base_ui_obj.check_element_text(
+            expected_text="Change connection details"
+        )
+        if conn_details:
+            logger.info("Click on Change Connection Details")
+            base_ui_obj.do_click(pvc_loc["connections-details"])
+        logger.info("Storage Class Service Name")
+        base_ui_obj.do_clear(pvc_loc["service-name"])
+        base_ui_obj.do_send_keys(pvc_loc["service-name"], "vault")
+        logger.info("Storage Class Address")
+        base_ui_obj.do_clear(pvc_loc["kms-address"])
+        base_ui_obj.do_send_keys(pvc_loc["kms-address"], "https://vault.qe.rh-ocs.com/")
+        logger.info("Storage Class Port")
+        base_ui_obj.do_clear(pvc_loc["kms-port"])
+        base_ui_obj.do_send_keys(pvc_loc["kms-port"], "8200")
+        logger.info("Click on Advanced Settings")
+        base_ui_obj.do_click(pvc_loc["advanced-settings"])
+        logger.info("Enter Backend Path")
+        base_ui_obj.do_clear(pvc_loc["backend-path"])
+        base_ui_obj.do_send_keys(pvc_loc["backend-path"], backend_path)
+        logger.info("Enter TLS Server Name")
+        base_ui_obj.do_clear(pvc_loc["tls-server-name"])
+        base_ui_obj.do_send_keys(pvc_loc["tls-server-name"], "vault.qe.rh-ocs.com")
+        logger.info("Enter Vault Enterprise Namespace")
+        base_ui_obj.do_clear(pvc_loc["vault-enterprise-namespace"])
+        base_ui_obj.do_send_keys(pvc_loc["vault-enterprise-namespace"], namespace)
+        logger.info("Selecting CA Certificate")
+        base_ui_obj.do_click(pvc_loc["browse-ca-certificate"])
+        time.sleep(1)
```

Comment:
Do you must to use `sleep`?

Reply:
Yes, Since I am using pyautogui to interact with the Linux console and upload the certificates, it takes some time to respond and that's why sleep has been added to avoid upload failures.